pkg/cloudprovider/types.go

```diff
@@ -17,7 +17,7 @@ package cloudprovider
 import (
 	"context"
 
-	provisioningv1alpha1 "github.com/awslabs/karpenter/pkg/apis/provisioning/v1alpha1"
+	"github.com/awslabs/karpenter/pkg/apis/provisioning/v1alpha1"
```

Comment:
why is the alias removed here? Seems clearer to keep 

```
provisioning "github.com/awslabs/karpenter/pkg/apis/provisioning/v1alpha1"
```

Reply:
When possible, I don't alias my packages. If we do need to alias them, I'm fine with dropping the version from the alias.